app/views/course/groups/_users.html.slim

```diff
@@ -0,0 +1,14 @@
+table.table
```

Comment:
I think `_users` is easier for others to understand ?  do we have any conventions here ?